### PR TITLE
fix regression Reaction Ensemble

### PR DESCRIPTION
**Pull request by jonaslandsgesell**
Fixes regression from changing from K_P to gamma=K_c^ideal

### CODE REVIEW

File: src/python/espressomd/c_analyze.pxd [hunk 1]
```diff
@@ -98,6 +98,7 @@ cdef extern from "statistics_chain.hpp":
     void calc_re(PartCfg & , double ** re)
     void calc_rg(PartCfg & , double ** rg)
     void calc_rh(PartCfg & , double ** rh)
+    void analyze_rdfchain(PartCfg &, double r_min, double r_max, int r_bins, double ** f1, double ** f2, double ** f3)
```
Comment (jonaslandsgesell) [inline src/python/espressomd/c_analyze.pxd]: this comes from statistics_chain.hpp

Comment (fweik) [reply src/python/espressomd/c_analyze.pxd]: This function has been removed.

**Review by fweik — APPROVED** [review]

**Comment by KaiSzuttor** [comment]
d’oh, merged too early

**Comment by jonaslandsgesell** [comment]
I did not touch those parts though.what about running the scripts
automatically on 4.0?

Kai Szuttor <notifications@github.com> schrieb am Mo., 3. Sep. 2018, 18:37:

**Pull request merged**